clients/web/src/lib/span/update/kinds/ipc/ipc-spans.ts

```diff
@@ -0,0 +1,11 @@
+type IpcKind = "ipc";
+
+export const IpcKinds = [
+  {
+    type: "ipc",
+    names: new Set(["wry::ipc::handle", "wry::custom_protocol::handle"]),
+  },
+] as const satisfies {
```

Comment:
if you type it as satisfying the constraints without readonly, you can just drop the const assertion, since it means the same.

Reply:
Good to know 👍